src/api/auth/auth.js

```
@@ -1,8 +1,104 @@
+    }
+    
+    localStorage.setItem('fcm_initialized', 'true');
+  } catch (error) {
+    console.error("Service Worker registration failed:", error);
+  }
+};
+
+// FCM 토큰 등록 함수
+export const registerFCMToken = async (memberId) => {
+  if (!memberId) {
+    console.error("Member ID is required");
```

ESLint

[error] Replace `"Member·ID·is·required"` with `'Member·ID·is·required'`

(prettier/prettier)